articles/install-guide/index.md

```diff
@@ -1,53 +1,225 @@
+
+    ```bash
+    conda install -c quantum-engineering iqsharp
+    ```
+
+1. Verify the installation by creating a `Hello World` application
+
+    - Run the following command to start the notebook server:
+
+        ```bash
+        jupyter notebook
+        ```
+
+    - Browse to the URL shown on the command line. For example: http://localhost:8888
+
+    - Create a jupyter notebook with a Q# kernel, and add the following code to the first notebook cell:
+
+        ```qsharp
+        operation HelloQ () : Unit {
+            Message($"Hello from quantum world!");
+        }
+        ```
+
+    - Run this cell of the notebook:
+
+        You should see `HelloQ` in the output of the cell. When running in jupyter notebooks, the Q# code is compiled, and the notebook outputs the name of the operation(s) that it finds.
+
+## Develop with C# on Windows, using Visual Studio
+
+1. Pre-requisites
+
+    - Visual Studio, version, with the .NET Core cross-platform development workload enabled
+
+1. Install the Q# Visual Studio extension
+
+    - Download the [Visual Studio extension](https://marketplace.visualstudio.com/items?itemName=quantum.DevKit)
+    - Add the extension to Visual Studio
+
+1. Verify the installation by creating a `Hello World` application
+
+    - Create a new Q# application
+
+        - Go to **File** -> **New** -> **Project**
+        - Type `Q#` in the search box
+        - Select **Q# Application**
+        - Select **Next**
+        - Choose a name and location for your application
+        - Select **Create**
+
+    - Inspect the project
+
+        You should see that two files have been created: `Driver.cs` which is the C# host application; and `Operation.qs`, which is a Q# program that defines a simple operation to print a message to the console.
+
+    - Run the application
+
+        - Select **Debug** -> **Start Without Debugging**
+        - You should see the text `Hello quantum world!` printed to a console window.
+
+## Develop with C#, using VS Code
+
+1. Pre-requisites
+
+   - [VS Code](https://code.visualstudio.com/download)
+   - [.NET Core SDK 2.1 or later](https://www.microsoft.com/net/download)
+
+1. Install the Quantum VS Code extension
+
+    - Download the [VS Code extension](https://marketplace.visualstudio.com/items?itemName=quantum.quantum-devkit-vscode)
+    - Add the extension to VS Code
+
+1. Install the Quantum project templates:
+
+    ```bash
+    dotnet new -i Microsoft.Quantum.ProjectTemplates
+    ```
+
+    You now have the Quantum Development Kit installed and ready to use in your own applications and libraries.
+
+1. Verify the installation by creating a `Hello World` application
+
+    - Create a new project:
+
+        - Go to **View** -> **Command Palette**
+        - Select **Q#: Create New Project**
+        - Navigate to the location on the file system where you would like to create the application
+
+    - Open the new project:
+
+        - Back in VS Code, navigate to the new project
```

Comment:
The **Create new project...** command should offer to open the new project. Otherwise, I'd suggest emphasizing that the user needs to open the folder, not each individual file; that's been a significant point of confusion for people not used to Sublime-style open folder editors (e.g.: Sublime Text, Atom, VS Code, Visual Studio in anycode mode).

Reply:
Ah yes, that's right. I just re-tested it. Fixing